roles/nginx/files/nginx.conf

```diff
@@ -0,0 +1,63 @@
+user nginx nginx;
+worker_processes 2;
```

Comment:
I'd be interested to see what the `auto` setting does, but maybe that's a bad idea.